templates/democracy/readme.md

```diff
@@ -43,8 +41,8 @@ The network details will be automatically selected by the `arapp.json`'s environ
 
 ## Gas usage
 
-Tested running `GAS_REPORTER=true truffle test --network devnet test/gas.js`, plus `deploy-kit.js` script in `beta-base`.
+Tested running `GAS_REPORTER=true truffle test --network devnet test/gas.js`, plus `deploy.js` script.
 
-- Create the Kit:      2816197
+- Create template:     2816197
 - Create new token:    1738117
 - Deploy new instance: 5690035
```

Comment:
It would be nice to run these again and have updated values, to evaluate how the refactor will affect users.

Reply:
will create a new issue for the gas measurements